smart_card_connector_app/src/window.css

```diff
@@ -120,8 +112,8 @@ body {
   margin: 12px;
 }
 
-#add-device::after {
-  content: '__MSG_addDevice__';
+#app-list-header {
+  margin: 0px 0 0;
```

Comment:
Why did this become necessary?
I think the whole layout was already approved with the UI team.

(And, if this line is actually needed, then please change it to the shortest format: "margin: 0".)